tuftool/tests/root_command.rs

```diff
@@ -84,7 +84,7 @@ fn add_key_all_roles(key: &str, root_json: &str) {
 fn sign_root_json(key: &str, root_json: &str) {
     Command::cargo_bin("tuftool")
         .unwrap()
-        .args(&["root", "sign", root_json, "-k", key])
+        .args(&["root", "sign", root_json, "-f", "-k", key])
```

Comment:
Can we update testing so we have cases with fewer than the required number of keys, one without the new flag that we ensure fails, and one with the new flag that we ensure passes?

Reply:
There is a test called `append_signature_root`, which initializes the root.json with root threshold 2, and uses `sign_root_json` with `-f`, which covers the the second part of the test that you ask. I'll the missing test case.